exp="10+20"

console.log('execption handling');

try{const result= eval(exp)
console.log("result is ",result);}

catch{console.log("error in line number 4");}

finally{console.log("programme complited");}

